src/Workspaces/Core/Portable/Rename/ConflictResolution.cs

```diff
@@ -38,7 +37,7 @@ namespace Microsoft.CodeAnalysis.Rename
         /// </summary>
         public readonly Solution? NewSolution;
 
-        public readonly bool? ReplacementTextValid;
+        public readonly bool ReplacementTextValid;
```

Comment:
i changed this to a bool since if the ConflictResolution is a failure, this is never read.  And if it's a non failure, it's only allowed to proceed if non-null and true.  So we can just literally have that be a simple boolean since we don't even care about it in the failure path.